src/main/java/be/seeseemelk/mockbukkit/plugin/PluginManagerMock.java

```diff
@@ -92,7 +92,7 @@ public void unload()
 		{
 			try
 			{
-				FileUtils.deleteDirectory(file);
+				FileUtils.forceDelete(file);
```

Comment:
Can you please elaborate on why this was changed?

Reply:
The previous method only deleted directories, and would fail if provided with a file. The thing is, now we are deleting both directories AND files. Instead of putting an if-else if, this method will delete both directories and files (in fact, if provided a directory, it will call the previously used `deleteDirectory`). If needed, I can convert this to an if-else if, but I would find this not necessary, as `forceDelete` is already doing exactly what we need here.